cvmfs/network/download.cc

```diff
@@ -70,6 +70,45 @@ using namespace std;  // NOLINT
 
 namespace download {
 
+/**
+ * Returns the status if an interrupted happened for a given repository.
+ *
+ * Used only in case CVMFS_FAILOVER_INDEFINITELY (failover_indefinitely_) is set
+ * where failed downloads are retried indefinitely, unless an interrupt occurred
+ *
+ * @note If you use this functionality you need to change the source code of
+ * e.g. cvmfs_config reload to create a sentinel file. See comment below.
+ *
+ * @return true if an interrupt occurred
+ *         false otherwise
+ */
+bool Interrupted(const std::string &fqrn, JobInfo *info) {
+  if (info->allow_failure()) {
+    return true;
+  }
+
+  if (!fqrn.empty()) {
+    // see https://github.com/cvmfs/cvmfs/pull/3304#discussion_r1278991197
```

Comment:
```suggestion
```
I'd remove this and expand the comment if needed - better for readability if there are no external links.